### DISCUSSION TITLE
Warning when importing module with rollup

### DISCUSSION THREAD

**Discussion by CameronNemo** [enisdenjo/graphql-ws#443] [Q&A] [answered]
```
(!) `this` has been rewritten to `undefined`
https://rollupjs.org/guide/en/#error-this-is-undefined
node_modules/graphql-ws/lib/server.mjs
4:  *
5:  */
6: var __asyncValues = (this && this.__asyncValues) || function (o) {
                        ^
7:     if (!Symbol.asyncIterator) throw new TypeError("Symbol.asyncIterator is not defined.");
8:     var m = o[Symbol.asyncIterator], i;
...and 1 other occurrence
```

**Expected Behaviour**

Import and bundle module without warnings with rollup. 

**Actual Behaviour**

Saw the above warning.

**Other**

https://rollupjs.org/troubleshooting/#error-this-is-undefined

**Comment by enisdenjo** [comment, marked as answer]
Hey there, the bundled code you're referencing comes from the TypeScript compiler. The only option we have here is to use the `importHelpers: true` option for TSC; however, this would introduce an additional dependency of `tslib` and increase the bundle size completely unnecessarily - this is something I don't intent to do.

The warning is actually safe to ignore, but if you insist on removing it - you should use the `context: "this"` option with rollup.

Read more:
- https://github.com/rollup/rollup/issues/3788
- https://github.com/developit/microbundle/issues/682
